helmenv.sh

```diff
@@ -3,16 +3,24 @@
 function _helmenv_test_requirements {
     if [[ ! "$(command -v curl)" ]]
     then
-        echo "You must install curl"
-        exit 1
+        echo "helmenv: You must install curl"
+        exit 0
```

Comment:
Changing the exit status since `exit 1` was killing `bash` in addition to the script.

This made it difficult to figure out that, on initial installation, `jq` needed to be installed because the `bash` process which had the error message would immediately close after the script exited.